### PR TITLE
Fix title font size for small screen devices

### PR DESCRIPTION
**Pull request by utkarsh23**
Resolved issue #23 

after

### CODE REVIEW

**Comment by newbazz** [comment]
Write the "Closes <issue link>" in the commit.

**Review by danishprakash — COMMENTED** [review]
Always read [coala commit writing guidelines](http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html) before commiting. Kindly change your commit according to the guidelines.
Until then, wait for someone with appropriate permissions to review the PR. 👍

File: coala.css [hunk 1]
```diff
@@ -85,6 +85,12 @@ nav {
   margin-top: 4em;
 }
 
+@media screen and (max-width: 992px) {
+  .title {
+    font-size: 11em;
+  }
+}
+
```
Comment (jayvdb) [inline coala.css]: we dont want two blank lines here.

**Review by jayvdb — CHANGES_REQUESTED** [review]

**Comment by utkarsh23** [comment]
@jayvdb its fixed

**Review by jayvdb — CHANGES_REQUESTED** [review]
Please squash your commits.

**Comment by utkarsh23** [comment]
@jayvdb squashed the two commits :+1:

**Review by newbazz — COMMENTED** [review]
Use "Fixes" instead of "Closes" in commit message.

**Comment by yashovardhan** [comment]
Your code looks fine, but please follow the [commit guidelines](http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html). Your commit description's first line is too long, please reduce the number of characters in the first line as according to commit guidelines only 72 characters are allowed per line.

**Review by yashovardhan — CHANGES_REQUESTED** [review]
unack https://github.com/coala/coalaCSS/pull/24/commits/334696fd677149e8a0e06dcfe48d27b4878bb8f0

**Comment by Monal5031** [comment]
@utkarsh23 Use `Closes` in the commit message body.

**Comment by jayvdb** [comment]
unack https://github.com/coala/coalaCSS/pull/24/commits/7e57de9428552990f2ba5d801b4e59d80df64b0e

**Comment by utkarsh23** [comment]
Is there more work to be done on this PR? I believe I've fixed everything @jayvdb

**Review by realslimshanky — CHANGES_REQUESTED** [review]
Your commit message has a shortlog of 55 characters which should be a maximum of 50. Please follow [this](https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html) documentation on writing a good commit message. Amend your commit and push it again.

**Review by Monal5031 — APPROVED** [review]
LGTM :+1:

**Comment by Monal5031** [comment]
ack de3776e

**Review by realslimshanky — APPROVED** [review]

**Comment by hemangsk** [comment]
@gitmate-bot rebase

**Comment by hemangsk** [comment]
@rultor merge 🎉

**Pull request merged**